tensorboard/pip_package/pip_smoke_test.sh

```diff
@@ -170,5 +189,9 @@ rm -rf "${PIP_TMP_DIR}"
 rm -rf "${TMP_LOGDIR}"
```

Comment:
I should have mentioned this in the original PR, but—is the `-f` needed here? We should prefer to use `-r` alone if possible.